### DISCUSSION TITLE
Any idea about which hack?

### DISCUSSION THREAD

**Discussion by SnowmanDK** [roleoroleo/yi-hack-Allwinner-v2#910] [Q&A] [answered]
I have bought a camera, and would like to hack it, if possible.
The problem is, I don't know which firmware to use (if any is compatible).
Can anyone help me with a pointer?
![IPCam](https://github.com/user-attachments/assets/d4fb32a5-ad0d-4e20-8d60-6cfa765441c1)

**Comment by SnowmanDK** [comment]
I forgot to mention, that alienatedsec (yi-hack-v5 author) recomended that I looked here at Allwinner-v2, but I am not sure which hack to use (if any).

**Comment by roleoroleo** [comment]
Sorry but Anyka paltform is not supported by my hacks.

**Comment by SnowmanDK** [comment, marked as answer]
Just to let others know...
I found a working hack for my camera here: [https://github.com/MuhammedKalkan/Anyka-Camera-Firmware](https://github.com/MuhammedKalkan/Anyka-Camera-Firmware)
